plugins/woocommerce/templates/single-product/add-to-cart/simple.php

```diff
@@ -46,7 +46,7 @@
 		do_action( 'woocommerce_after_add_to_cart_quantity' );
 		?>
 
-		<button type="submit" name="add-to-cart" value="<?php echo esc_attr( $product->get_id() ); ?>" class="single_add_to_cart_button button alt"><?php echo esc_html( $product->single_add_to_cart_text() ); ?></button>
+		<button type="submit" name="add-to-cart" value="<?php echo esc_attr( $product->get_id() ); ?>" class="single_add_to_cart_button button alt<?php echo esc_attr( wc_wp_theme_get_element_class_name( 'button' ) ? ' ' . wc_wp_theme_get_element_class_name( 'button' ) : '' ); ?>"><?php echo esc_html( $product->single_add_to_cart_text() ); ?></button>
```

Comment:
@peterfabian is there a need to call `wc_wp_theme_get_element_class_name( 'button' )` twice? Could this be

```
<?php echo esc_attr( wc_wp_theme_get_element_class_name( 'button' ) ); ?>
```

Reply:
@helgatheviking It would need a leading space but that could be added after `alt`.

Reply:
Gotcha. Seems like worse case is there's a trailing space if you remove the duplication...  `class="single_add_to_cart_button button alt ">`